packages/tools/src/lib/build-cache/fetchCachedBuild.ts

```diff
@@ -66,6 +66,15 @@ To disable this warning, set the provider to null:
 
   loader.stop(`No local build cached. Checking ${remoteBuildCache.name}.`);
 
+  const availabilityState = await remoteBuildCache.getAvailabilityState();
+  if (!availabilityState.isAvailable) {
+    logger.warn(
+      `${remoteBuildCache.name} caching is not available: ${availabilityState.reason}`
+    );
+
+    return null;
+  }
```

Comment:
I think I would like to avoid that. What to do instead, is to throw error, which we'll then catch and display as a warning, giving user possibility to choose whether to continue with local build or not: https://github.com/callstack/rnef/pull/359